index.js

```diff
@@ -0,0 +1,31 @@
+var calculator = new Vue({
+    el: '#calculator',
+    data: {
+        price: '',
+        tipRate: '20',
+        total: ''
+    },
+    computed: {
+        numTipTotal: function(e) {
+          var percentage = this.tipRate/100 + 1;
+          var calculatedTotal = percentage * this.price
+          return calculatedTotal
+        },
+        calcTotal: function(e){
+            return currencyFormat(this.numTipTotal)
+        },
+        numFormat: function(e){
+            e.replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1,");
+        },
+        calcTip: function(e){
+          var calculatedTip = this.numTipTotal - this.price;
+          return currencyFormat(calculatedTip)
+        }
+    }
+
+});
+
+
+function currencyFormat (num) {
+    return "$" + num.toFixed(0).replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,")
```

Comment:
this is nitpicky, but is there a reason you rounded?  As a server, I appreciate the rounding since i hate dealing with coins, but just curious why you chose not to put 2 instead of 0 here.
Also, not quite sure what the rest of the line actually does, I see stuff that looks like escape "/" marks, but not sure what its actually doing.